homeassistant/components/notify/apns.py

```diff
@@ -0,0 +1,289 @@
+"""
+APNS Notification platform.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/notify.apns/
+"""
+import logging
+import os
+import voluptuous as vol
+
+from homeassistant.helpers.event import track_state_change
+from homeassistant.config import load_yaml_config_file
+from homeassistant.components.notify import (
+    ATTR_TARGET, ATTR_DATA, BaseNotificationService)
+import homeassistant.helpers.config_validation as cv
+from homeassistant.helpers import template as template_helper
+
+DOMAIN = "apns"
+APNS_DEVICES = "apns.yaml"
+DEVICE_TRACKER_DOMAIN = "device_tracker"
+SERVICE_REGISTER = "register"
```

Comment:
Can you rename it `apns_register` to indicate that it is part of the apns platform?